scipy/ndimage/tests/test_datatypes.py

```diff
@@ -52,17 +52,17 @@ def test_uint64_max():
     # Test interpolation respects uint64 max.  Reported to fail at least on
     # win32 (due to the 32 bit visual C compiler using signed int64 when
     # converting between uint64 to double) and Debian on s390x.
-    # Interpolation is always done in double precision floating point, so we
-    # use the largest uint64 value for which int(float(big)) still fits in
+    # Interpolation is always done in double precision floating point, so
+    # we use a larges uint64 value for which int(float(big)) still fits in
```

Comment:
"larges"? I like the original better here.

Reply:
It looks like there were some numerical stability issues with the way I had implemented things that were making this test misbehave. I have gone back to having a first loop applying a `gain` to all coefficients, rather than doing it during the anticausal filtering, and it all works again.